h2o-algos/src/main/java/hex/gam/GamSplines/ThinPlateRegressionUtils.java

```diff
@@ -0,0 +1,347 @@
+package hex.gam.GamSplines;
+
+import water.MRTask;
+import water.fvec.Chunk;
+import water.fvec.Frame;
+import water.fvec.NewChunk;
+import water.util.ArrayUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static hex.gam.GAMModel.GAMParameters;
+import static water.util.ArrayUtils.maxValue;
+
+// This class contains functions that perform different functions for generating the thin plate regression splines
```

Comment:
could be javadoc

Reply:
Done.

Reply:
@wendycwong, I cant see it is Javadoc, am I missing something?